docs: need to enable `disableImportExportTransform` for tree shaking (#1491)
microsoft/rnx-kit: docsite/docs/guides/bundling.mdx

}
```

> In order for tree shaking to work properly, we will also need to enable
> `disableImportExportTransform` in our Babel config:

Reviewer: ```suggestion
In order for tree shaking to work properly, you need to enable
`disableImportExportTransform` in your Babel config:
```

Reviewer: Thanks for catching this!
